src/esp/bindings/SensorBindings.cpp

```diff
@@ -54,7 +55,11 @@ void initSensorBindings(py::module& m) {
   py::enum_<SensorSubType>(m, "SensorSubType")
       .value("NONE", SensorSubType::None)
       .value("PINHOLE", SensorSubType::Pinhole)
-      .value("ORTHOGRAPHIC", SensorSubType::Orthographic);
+      .value("ORTHOGRAPHIC", SensorSubType::Orthographic)
+      .value("FISHEYE", SensorSubType::Fisheye);
```

Comment:
@bigbike Not sure this sensorsubtype is necessary right? I just overloaded the sensor_subtype enum on FishEyeSensorSpecs with the FisheyeSensorModelType.

Reply:
Yes, I believe we need the sub-type of the sensor. You are doing the correct thing here.

Reply:
Yeah, but we don't end up using it because I am overriding SensorType to be the modelType for the FishEye Sensor, should I avoid doing that?

Reply:
Correct, if I were you, I would avoid doing that. This causes inconsistency between the sim engine and the py interface.
Also in terms of design, I think it is enough at this level that user understands it is a fisheye sensor (without knowing the actual camera model). 
And if the user does not set the actual camera type, by default, the simulator will adopt "doubleSphere" model (assuming we have a couple of options in the future).